README.md

```diff
@@ -8,11 +8,13 @@ Our primary goal is to streamline versioned data operations in lakeFS, enabling
 
 Highlights:
 
-- High-level abstraction over basic lakeFS repository operations
-- Seamless integration into the fsspec ecosystem
-- Transaction support
-- Zero-config option through config autodiscovery
-- Automatic up-/download management to avoid unnecessary transfers for unchanged files
+- Simple repository operations in lakeFS
+- Easy access to underlying storage and versioning operations
+- Seamless integration with the fsspec ecosystem
+- Directly access lakeFS objects from popular data science libraries (including Pandas, Numpy, Polars, DuckDB, PyArrow) with minimal code
```

Comment:
Does numpy really offer fsspec support? I wasn't able to find any mention of that.

Reply:
Fixed